### PR TITLE
Changed accessibility to login, added account settings form.

### PR DESCRIPTION
**Pull request by Azuolas3**
- More intuitive/logical UI for logging on and off.
- Account settings form
- Basic input validation for forms

### CODE REVIEW

File: ClientApp/src/components/AccountSettings.jsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+              <Input
+                type="checkbox"
+                onChange={(event) =>
+                  setPreferences({ ...preferences, soybean: event.target.checked })
+                }
+                value={preferences.soybean}
+              />
+              Soybean allergy
+            </Label>
+          </FormGroup>
```
Comment (onlyidev) [inline ClientApp/src/components/AccountSettings.jsx:241]: Code duplication can be reduced by extracting the creation of FormGroup component (can be the same file)

**Review by onlyidev — CHANGES_REQUESTED** [review]
Code looks good but AccountSettings component can be made much more readable by reducing code duplication

File: ClientApp/src/components/AccountSettings.jsx (hunk 1 above)
Comment (Azuolas3) [reply ClientApp/src/components/AccountSettings.jsx]: Good point, changed in next commit

Comment (Jurshenas) [reply ClientApp/src/components/AccountSettings.jsx]: Looks good

**Review by Jurshenas — APPROVED** [review]
Everything looks great

**Review by onlyidev — APPROVED** [review]

**Pull request merged**